examples/tutorials/vue.md

```diff
@@ -19,6 +19,17 @@ dinosaur page with more details. You can see the
 You can see a live version of the app on
 [Deno Deploy](https://tutorial-with-vue.deno.deno.net/).
 
+:::info Deploy your own
+
+Want to skip the tutorial and deploy the finished app right now? Click the
+button below to instantly deploy your own copy of the complete SvelteKit
+dinosaur app to Deno Deploy. You'll get a live, working application that you can
+customize and modify as you learn!
+
+[![Deploy on Deno](https://deno.com/button)](https://app.deno.com/new?clone=https://github.com/denoland/tutorial-with-vue&mode=dynamic&entrypoint=api/main.ts&build=deno+task+build&install=deno+install)
```

Comment:
The button params in the repos's readme don't match these. I guess they need an update too